CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 - Added AbstractAdmin, replacing Admin
 - Added `BaseMapper::keys` method
+- Added additional _ALL role check to RoleSecurityHandler 
```

Comment:
This line is not placed at the right section. Please move it on `[3.x]` one.

Or, just follow the new PR template to put it in: https://github.com/sonata-project/dev-kit/blob/master/project/.github/PULL_REQUEST_TEMPLATE.md#changelog